### PR TITLE
/security/certifications copyupdate

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates copy on /security/certifications
- Last minute copy changes on fips.
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/certifications
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against certification [copydoc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit)
- Compare against fips [copydoc](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit?ts=60f04150)
- Note: waiting on the link to the embedded video, see screenshot below.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4227

## Screenshots

Where the video should be:
image

### CODE REVIEW

**Comment by carkod** [comment]
Demo https://ubuntu-com-10036.demos.haus/security/certifications

File: templates/security/certifications.html [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
-     <p><a class="p-button--positive js-invoke-modal" href="/support/contact-us">Contact us</a></p>
+     <h1>Security Certifications & Hardening</h1>
+     <h2 class="p-heading--3">Run regulated and high security workloads on Ubuntu</h2>
+     <p>Whatever cybersecurity framework you have chosen, including ISO 27000, NIST, PCI or CIS Controls, <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> enable your compliance and reduce your operational risk. Access automation for hardening and compliance profiles, such as CIS and DISA-STIG as well as the FIPS 140-2 and Common Criteria certifications.</p>
+     <p><a class="p-button--positive js-invoke-modal" href="/security/contact-us">Contact us</a></p>
```
Comment (carkod) [inline templates/security/certifications.html:16]: Contact form is wrong, in the copy docs is `/security/certifications#get-in-touch`

File: templates/security/certifications.html [hunk 2]
```diff
@@ -35,33 +35,43 @@ <h2 class="p-heading--3">Deploy regulated and high security workloads</h2>
...
  </div>
 </section>
 
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>FIPS certification and CIS compliance with Ubuntu</h2>
+      <p>Learn about Ubuntu CIS and FIPS certified components to enable operating under compliance regimes like FedRAMP, HIPAA, PCI and ISO. Get all of your compliance questions answered in our upcoming webinar to ensure you and your team are, and remain, compliant.</p>
+      <p><a class="p-button--positive js-invoke-modal" href="/security/contact-us">Contact us</a></p>
+    </div>
```
Comment (carkod) [inline templates/security/certifications.html:60]: Same contact form issue. It is not the right modal.

File: templates/security/certifications.html [hunk 3]
```diff
@@ -35,33 +35,43 @@ <h2 class="p-heading--3">Deploy regulated and high security workloads</h2>
...
    </div>
    <div class="col-4 p-divider__block">
-     <h3>Transparency</h3>
-     <p>Canonical is transparent on the Security Features we enable on Ubuntu. We have a <a href="/security/notices">public vulnerability disclosure policy</a> and known vulnerabilities are disclosed in <a href="/security/oval">Machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party tools.</p>
+     <h3>Know your defenses</h3>
+     <p>Each Ubuntu release enables state of the art protection against vulnerability exploitation and malware and we <a class="p-link--external" href="https://wiki.ubuntu.com/Security/Features">publicly detail our choices</a>. Canonical has a <a href="/security/notices">public vulnerability disclosure policy</a> and vulnerabilities are not only fixed with automated security updates and <a href="/security/livepatch">livepatches</a> but also publicly disclosed with our <a href="/security/notices">security notices</a>. We further provide <a href="/security/oval">machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party vulnerability management tools.</p>
```
Comment (carkod) [inline templates/security/certifications.html:43]: "public vulnerability disclosure policy" link is incorrect. In copy docs it's `/security/disclosure-policy`

File: templates/security/certifications.html [hunk 4]
```diff
@@ -107,15 +117,15 @@ <h2>Ubuntu compliance profiles</h2>
...
+       <small>Certified tooling & automation</small>
      </li>
      <li class="p-inline-list__item">
        <img src="https://assets.ubuntu.com/v1/2ccda8d7-tick-orange.svg" alt="Yes: STIG content" style="height: 14px; vertical-align: middle; width: 14px;">
-       <small> STIG content</small>
+       <small> STIG guide</small>
      </li>
    </ul>
-   <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+   <a href="/security/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
```
Comment (carkod) [inline templates/security/certifications.html:128]: Wrong form

File: templates/security/certifications.html [hunk 5]
```diff
@@ -178,7 +178,7 @@ <h2>Ubuntu security certifications</h2>
        <small>Tooling & automation</small>
      </li>
    </ul>
-   <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+   <a href="/security/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
```
Comment (carkod) [inline templates/security/certifications.html:181]: wrong form

File: templates/security/certifications.html [hunk 6]
```diff
@@ -190,7 +190,7 @@ <h3>How do I harden my Ubuntu system?</h3>
...
    <p>Achieving and maintaining compliance is a complex and costly process that involves business processes in addition to software requirements. Ubuntu by Canonical contains software and security controls, such as disk encryption, password settings configuration, <a href="/security/fips">cryptographic compliance with FIPS140-2</a>, CIS hardening as well as <a href="/advantage">a comprehensive Enterprise software maintenance program</a>, to achieve and maintain compliance with the standard.</p>
-   <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+   <a href="/security/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
```
Comment (carkod) [inline templates/security/certifications.html:193]: wrong form

File: templates/security/certifications.html [hunk 7]
```diff
@@ -35,33 +35,43 @@ <h2 class="p-heading--3">Deploy regulated and high security workloads</h2>
...
+     <h3>Access certifications for high security environments</h3>
      <p>Access to certification artifacts as well as the necessary tooling for regulated and high security environments. <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a> provide access to FIPS 140-2 certified cryptographic packages, allowing you to deploy workloads that need to operate under compliance regimes like FedRAMP, HIPAA, and PCI-DSS. Additionally, Ubuntu versions have been certified under Common Criteria, providing 3rd party attestation of the security mechanisms in the operating system.</p>
      <a class="p-button--neutral" href="#security-certifications">See our certifications</a>
    </div>
  </div>
 </section>
 
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
```
Comment (carkod) [inline templates/security/certifications.html:56]: Is there a reason we only use half of the space?

File: templates/security/certifications.html (hunk 1 above)
Comment (anthonydillon) [reply templates/security/certifications.html:16]: Don't link to `#get-in-touch` this is purely a JS interaction that is handled via the `js-invoke-model` class. The link should link to a backup static form for non-JS.

File: templates/security/certifications.html (hunk 7 above)
Comment (anthonydillon) [reply templates/security/certifications.html:56]: This should probs have a design needed label

Comment (anthonydillon) [reply templates/security/certifications.html:56]: > Is there a reason we only use half of the space?

There should be a video in the other half I assume.

File: templates/security/certifications.html (hunk 1 above)
Comment (petesfrench) [reply templates/security/certifications.html:16]: I spoke to the pm about it but I think there was a misunderstanding, so in the end just linked to security/contact-us as it seemed most appropriate.

File: templates/security/certifications.html (hunk 7 above)
Comment (petesfrench) [reply templates/security/certifications.html:56]: Yes I was in discussion about what the PM wanted, added now.

Comment (petesfrench) [reply templates/security/certifications.html:56]: Yes I was in discussion about what the PM wanted, added now.

**Review by carkod — APPROVED** [review]
Looks good now!

**Pull request merged**